web/client/src/styles/globals.css

```
@@ -0,0 +1,53 @@
+@tailwind base;
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)